include/arch/x86/linker.ld

```diff
@@ -216,6 +229,23 @@ SECTIONS
 	}
 #endif
 
+#ifdef CONFIG_MMU
+	/* Memory management unit*/
+	SECTION_PROLOGUE(mmulist, (OPTIONAL),)
+	{
+	/* get size of the mmu lists needed for gen_mmu.py*/
+	LONG((__MMU_LIST_END__ - __MMU_LIST_START__) / __MMU_REGION_SIZEOF)
+	/* Get the start of mmu tables in data section so that the address
+	 * of the page tables can be calculated.
+	 */
+	LONG(__mmu_tables_start)
+	__MMU_LIST_START__ = .;
+	KEEP(*(.mmulist))
+	KEEP(*(.gnu.linkonce.mmulist))
```

Comment:
Since this is a draft version I kept some of these things. 
Sure I will remove them, I meant to remove it in the final version.